### ISSUE TITLE
No save changes

### ISSUE THREAD

**Issue by zhuraavl** [michelson/Dante#103]
Excuse me for asking, but I'm beginner.
I have added your beautiful plugin to my project and can not customize the preservation of records. Currently my code form in view file looks like this:
```
<%= form_for :post, url: posts_path do |f| %>
  <% if @post.errors.any? %>
    <div id="errors">
      <p>
        <%= pluralize(@post.errors.count, "error") %> prevented from saving
      </p>
      <ul>
        <% @post.errors.full_messages.each do |msg| %>
        <li><%= msg %></li>
        <% end %>
      </ul>
    </div>
  <% end %>
  <p>
    <%= f.label :title %> <br>
    <%= f.text_field :title %>
  </p>
  <p>
    <span id="editor" > <%= @post.body.try(:html_safe) %> </div>
  </p>
  <p>
    <%= f.submit %>
  </p>
<% end %>
<script>
editor = new Dante.Editor(
    {
      el: "#editor",
      upload_url: "/images.json", //it expect an url string in response like /your/server/image.jpg or http://app.com/images/image.jpg
      store_url: "/posts/new" //post to save
    }
  );
  editor.start()
  editor.getContent()
</script>
```
the controller and the model has nothing unusual. I try to make a blog

If i try to submit a form, an error that the field Body can't be blank.

Also, please tell me if I can make it work, to upload pictures to the server I need to be some additional setup or gem?

Now I tried this code and it worked. However, attempts to edit the record does not succeed, the application freezes or simply inserts the necessary html not edit zone, and just down.
```
$( "#submit-yes" ).click(function() {
    form = $("#form");
    form.find("#body-wrap").val( editor.getContent() );
  });
```

If the recording is a picture, the application hangs.
I think this is not the best way.

**Comment by michelson** [comment]
you could bind the submit event of the form:

```
form.on("submit", function(e){
   form.find("some_text_field").val( form.find("#post").val( editor.getContent() ) )
})
```

or 
```
$( "#submit-yes" ).click(function(ev) {
    ev.preventDefault()
    form.find("#body-wrap").val( editor.getContent() );
    form.submit();
  });
```

**Comment by zhuraavl** [comment]
I tried to paste your code, but nothing happened. Moreover, the text stored ceased.

Code which I have cited above, it is saved in the second time. Updating also occurs in the second time

However, if I try to edit the record containing the image, the application freezes and my mac heats up and begins to make noise. Apparently he is as tired as I am))

I will try to create a new project and add to the editor. Because I have a suspicion that they do not work
